### ISSUE TITLE
Links in the Wiki outdated

### ISSUE THREAD

**Issue by ashtif** [rathena/rathena#6269]
I finished setup for the server side of things, however when I went to move onto the client patching, not only did the instructions seem quite unclear, the link provided for the data folder was broken.

I've also tried using an alternative guide for patching the client, but that information was outdated as well.

I'm used to be quite experienced with setting up a client about 5-6 years ago, but I can't seem to find a good place with solid instructions and good resources, aside from the server setup guide provided here.

Thanks.

**Comment by ashtif** [comment]
Additionally, there's a very odd issue in my server, where immediately after sailing to Izlude after the starter quest, I spawned in the water on the northeastern side next to the captain, a doram and another NPC. Had to use the `@go` command to get into the actual town, then I kept running into invisible objects/walls in the town on my way to finish the starter quest.

Running renewal.

**Comment by karthikavinash** [comment]
The guides are on rathena forums. They are dated but still work. Most of the server and client setup topics are covered on the forums.

**Comment by ashtif** [comment]
Thanks for the quick response. Any chance you know if I messed up with server setup from the starting Izlude in the ocean, unable to move issue?

**Comment by karthikavinash** [comment]
It could be your client setup. Which data files are you using?

**Comment by ashtif** [comment]
Client version is 2020 06 03, data files are the ro translation data files for Renewal from @ROenglishRE and the kRO full client 2020 06 03 Sakray / RSU patch

**Comment by ashtif** [comment]
And the patching guide I went with was [Renzadi'c Guide](https://rathena.org/board/guides/client-guides/guide-setting-up-2020-client-adding-customs-items-r4/)

**Comment by ashtif** [comment]
One thing I found odd about Renzadic's guide was using both the pre-re patch and the re patch for renewal, in the order of using pre-re first followed by using the renewal patch. I thought that could be part of the problem.

**Comment by karthikavinash** [comment]
This isn't an issue. It can't be replicated and also the problem is improper client setup. So, please close this issue. You can ask for help in the forums or on the discord server.

**Comment by ashtif** [comment]
So as far as I can tell, this isn't a client issue.

I've found the problem with the map collision being off, it's actually that `map_cache.dat` in `db\re` doesn't have the newest version of `izlude.gat`

I'm not sure if there was an event that superseded the original .gat for Izlude, but now that I've solved that, I am now having the issue of no portals appearing in Izlude, nor are the NPCs in the correct places.

I'm running Renewal for the server, as I said, and if this is a client issue, there isn't any documentation that I can find regarding how to get NPCs and portals to appear in the correct places.

Thanks.
